app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] As mentioned by the previous reviewers, kindly make sure the user is able to navigate to your application. Right now when a user tries to **Sign Up** the **Confirmation** action doesn't work so impossible to navigate to the whole application without being authenticated.

Reply:
Hi @tresorsawasawa 
You can find the verification link in the console.

Reply:
https://user-images.githubusercontent.com/18054628/188271770-9b29d5e3-20b1-45f7-87fc-ed16e7e9f1c6.mp4

Reply:
Hi @devMahmoud 

Yes, I get it but I think this is supposed to be done automatically as every user is not supposed to know about programming. 
Imagine you deploy your project online, how will the user be able to **Sign Up**?

Reply:
Hi @tresorsawasawa 
For development, It's ok to have the verification link in the console.
Also, the requirements didn't specify how to verify the user account.

Reply:
Hi @devMahmoud 

You are true. I am really sorry for that. You can make another request review

Reply:
Ok, I will ask for another review
thanks